module3-crud-and-data-models/r1-schema-and-data-models/README.md

```diff
@@ -0,0 +1,103 @@
+## Intoduction to database schemas
+
+A database schema is a blueprint or architecture of how our data will look. It doesn’t hold data itself, but instead describes the shape of the data and how it might relate to other tables or models.
+
+For example, this is a `customer` schema in MongoDB using mongoose.
+
+```js
+const mongoose = require("mongoose");
+
+const customer = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  zipcode: {
+    type: Number,
+  },
+});
+
+module.exports = mongoose.model("Customer", customer);
+```
+
+The same `customer` schema in a MySql database.
+
+```sql
+CREATE TABLE customer (
+ id INT AUTO_INCREMENT PRIMARY KEY,
+ name VARCHAR(50) NOT NULL,
+ zipcode INT(15) default NULL,
+)
+
+```
+
+We can see that the schema definition for the `customer` has a `name` which is a type `string` and a `zipcode` that is of type `number`.
+
+### The Difference between data model and schema
+
+The database schema is one that contains list of attributes and instructions to tell the database engine how data is organised whereas data model is a collection of conceptional tools for describing data, data-relationship and consistency constraints.
```

Comment:
This needs an example to be well understood. Data modeling could be related to multiple objects or resources. While schema is for one object shape in the database.

Reply:
Agree, it needs an example, however, looking at this now doesn't seem right to be here, I think it's already similar to what is in module 2 as Louis said above, I will just remove it from here.